### ISSUE TITLE
Error message"Colmap camera model not handled!"

### ISSUE THREAD

**Issue by henrypearce4D** [graphdeco-inria/gaussian-splatting#4]
Error message;

```
(gaussian_splatting) C:\gaussian-splatting>python train.py -s C:\gaussian-splatting\data\test
Optimizing
Output folder: ./output/15709e19-a [10/07 13:19:45]
Tensorboard not available: not logging progress [10/07 13:19:45]
Reading camera 1/31Traceback (most recent call last):
  File "train.py", line 204, in <module>
    training(lp.extract(args), op.extract(args), pp.extract(args), args.test_iterations, args.save_iterations)
  File "train.py", line 35, in training
    scene = Scene(dataset, gaussians)
  File "C:\gaussian-splatting\scene\__init__.py", line 44, in __init__
    scene_info = sceneLoadTypeCallbacks["Colmap"](args.source_path, args.images, args.eval)
  File "C:\gaussian-splatting\scene\dataset_readers.py", line 145, in readColmapSceneInfo
    cam_infos_unsorted = readColmapCameras(cam_extrinsics=cam_extrinsics, cam_intrinsics=cam_intrinsics, images_folder=os.path.join(path, reading_dir))
  File "C:\gaussian-splatting\scene\dataset_readers.py", line 95, in readColmapCameras
    assert False, "Colmap camera model not handled!"
AssertionError: Colmap camera model not handled!
```

Not sure what to do here, its a camera with 24mm lens, colmap 3.8 aligned the 31 image loop fine, any suggestions?

**Comment by henrypearce4D** [comment]
Hi, I downloaded the [T&T+DB Datasets (650MB)](https://repo-sam.inria.fr/fungraph/3d-gaussian-splatting/datasets/input/tandt_db.zip) and saw it was a colmap format so created my own.

I now see the section [Preprocessing your own Scenes](https://github.com/graphdeco-inria/gaussian-splatting#preprocessing-your-own-scenes) thanks for the heads up.

I just ran the convert.py and unless there is 'sparse' folder created manually next to the /input fodler the script will not run, could that info be added to the README.md or is that not correct?
